src/components/application_manager/src/application_data_impl.cc

```diff
@@ -436,6 +438,8 @@ void DynamicApplicationDataImpl::RemoveCommand(uint32_t cmd_id) {
     delete it->second;
     commands_.erase(it);
   }
+  auto& help_prompt_mngr = help_prompt_manager();
```

Comment:
@SNiukalov Const?

Reply:
No, using not constant method from object